tools/FakeItEasy.Build/Program.cs

```diff
@@ -112,8 +112,8 @@ public static void Main(string[] args)
 
         private static void RunTests(string testDirectory)
         {
-            var xml = Path.GetFullPath(Path.Combine(TestsDirectory, Path.GetFileName(testDirectory) + ".TestResults.xml"));
-            Run("dotnet", $"xunit -configuration Release -nologo -nobuild -noautoreporters -notrait \"explicit=yes\" -xml {xml}", testDirectory);
+            var xml = Path.GetFullPath(Path.Combine(TestResultsDirectory, Path.GetFileName(testDirectory) + ".TestResults.xml"));
+            Run("dotnet", $"test -c Release --no-build --logger:\"xunit;LogFilePath={xml}\"", testDirectory);

```

Comment:
There are many `dotnet xunit` options that don't have equivalent in `dotnet test`
- `-nologo`: I spent a bit of time trying to find it, but apparently there's no way to suppress the tool output header. I also didn't succeed in reducing the verbosity, even with `--verbosity quiet` (the default seems to be `minimal`, which produces the same output `quiet`)
- `-noautoreporters`: no equivalent, as far as I can tell. The tests are reported to AppVeyor again, and I don't know how to disable it.
- `-notrait`: there is a `--filter` option, but `--filter "explicit!=yes` doesn't work, even though the [docs mention it's possible](https://github.com/Microsoft/vstest-docs/blob/master/docs/filter.md). We're not using this at the moment anyway.

Reply:
For the first problem, I'd found https://github.com/Microsoft/vstest/issues/1701.
Ah, I forgot to check the auto reporters. Good find.

Reply:
> For the first problem, I'd found Microsoft/vstest#1701.

Yes, but it doesn't provide a solution...

Reply:
No, no. But it's nice to know that someone's aware of it.